static/audiofeed.yaml

```
@@ -1,3 +1,15 @@
+- title: 'Fantastische Fakten 1'
+  date: 2025-06-30T19:00
+  image: https://raw.githubusercontent.com/LucaNerlich/m10z/main/static/img/formate/cover/fakten.jpg
+  description: |2-
+                  Habt ihr ein Videospiel schon einmal gerochen? Oder vielleicht sogar schon einmal euer Leben aufs Spiel gesetzt, als ihr ein Game in die Konsole geladen habt?  Ihr wisst es nicht? In unserem neuen Format "Fantastische Fakten" könnt ihr es herausfinden.
+                  Jan, Til und Simon präsentieren euch absurde, verrückte, spannende und unglaubliche Fakten aus der Welt der Videospiele. 
+                  Hört mal rein, lasst euch überraschen und kommt auf <a href="https://community.wasted.de/">community.wasted.de</a> und stimmt für euren Favoriten ab.
+                  Welcher der drei Fakten hat euch am meisten aus dem Gaming Sessel gerissen.
+                  Wir sind gespannt auf euer Feedback
+  seconds: 1849
+  blogpost: https://m10z.de/ff-1
+  url: https://m10z.picnotes.de/FantastischeFakten/Fakten_001.mp3 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)